package.json

```diff
@@ -155,6 +156,7 @@
     "use-debounce": "^8.0.1",
     "uuid": "^8.3.2",
     "wcag-contrast": "^3.0.0",
+    "web-worker": "^1.2.0",
```

Comment:
What do you need web workers for?

Reply:
elkjs throws an error if this isn't installed. I think they forgot to include it as a dependency or something